cypress/support/helpers/getPageUrls.js

```diff
@@ -0,0 +1,18 @@
+const path = require('ramda/src/path');
+
+global.Cypress = { env: () => {} }; // Fake Cypress.env
+const services = require('../config/services');
+
+const getPageTypes = service => path([service, 'pageTypes'], services);
+
+const getPageUrls = ({ pageType, environment, isSmoke }) =>
+  Object.keys(services)
+    .map(getPageTypes)
+    .map(pageTypes => path([pageType], pageTypes))
+    .filter(config => (isSmoke ? config.smoke : true))
```

Comment:
Does this mean that it will only return smoke paths if we pass nothing in?

Reply:
Yes